unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarMeshCombiner/AvatarMeshCombinerUtils.cs

```diff
@@ -0,0 +1,329 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using DCL.Helpers;
+using UnityEngine;
+using UnityEngine.Rendering;
+using Object = UnityEngine.Object;
+
+namespace DCL
+{
+    public static class AvatarMeshCombinerUtils
```

Comment:
We should define a convention for `Utils` and ` Helpers` across the project.

Reply:
+1

Reply:
+1

For me right now it is be like this, but we may have to discuss it further:

Helper: A non-static utility class that serves a very self-containeed purpose.
Utils: A static utility class with no state.